float32 gps_x
float32 gps_y
int8 gps_status
int32 imu_yaw
string sign
string sign_traffic
string lane
string lidar
string lidar_p
string ctrl_mode
int32 gps_mission_index
int32 erp_speed
int32 erp_steering
int32 erp_break
int32 target_x
int32 target_y
